_content/monographs/karma-vibhangam_anandajoti.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Karma-vibhaṅgam"
+editor: "Sylvain Lévi"
+translator: anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Karma-vibhanga/Karma-vibhanga.pdf"
```

Comment:
The home page, not the PDF please: `https://ancient-buddhist-texts.net/Texts-and-Translations/Karma-vibhanga/`